SchibstedAccount/Core/GenericError.swift

```diff
@@ -20,16 +20,13 @@ extension GenericError {
 
 extension GenericError.Unexpected: CustomStringConvertible {
     var description: String {
-        if let string = self.string {
-            return "\(type(of: self))! \(string)"
-        }
-        return "\(type(of: self))!"
+        return "\(type(of: self))! \(string)"
```

Comment:
Even though it should never fail, can we avoid the bang here? Maybe something like `?? "n/a"`?

Reply:
The bang is part of the output text :p 

i.e. "Error! something failed"

Reply:
Oh, true! 😆